### PR TITLE
Move comment in system_files.py

### PR DESCRIPTION
**Pull request by cmcantalupo**

### CODE REVIEW

File: service/geopmdpy/system_files.py [hunk 1]
```diff
@@ -121,8 +121,8 @@ def _is_already_secure_directory(candidate_dir, perm_mode):
           is_secure_directory = True
     # Don't know what it is, definitely not a directory
     else:
+        # default condition: path is not a secure directory
```
Comment (tjs-intel) [inline service/geopmdpy/system_files.py]: I think it is most appropriate as a comment about line 113

Comment (cmcantalupo) [reply service/geopmdpy/system_files.py]: 👍

**Review by tjs-intel — APPROVED** [review]
I reviewed your spelling corrections as well, all looks good!

**Review by sidjana — APPROVED** [review]
LGTM!

**Pull request merged**